Data/IntSet/Internal.hs

```diff
@@ -290,11 +290,11 @@ instance Semigroup IntSet where
 
 instance Data IntSet where
   gfoldl f z is = z fromList `f` (toList is)
-  toConstr _     = fromListConstr
-  gunfold k z c  = case constrIndex c of
+  toConstr _    = fromListConstr
```

Comment:
It tries to align adjacent definitions if the added whitespace is under some threshold. This can be customized or turned off entirely. See [1] for the available options and let me know which one you would prefer.

[1] https://github.com/lspitzner/brittany/blob/4ded834a97c99979885b06ecd12ed62f61896884/src/Language/Haskell/Brittany/Internal/Config/Types.hs#L208